index.html

```diff
@@ -3,44 +3,129 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <script src="https://kit.fontawesome.com/39da8831d4.js" crossorigin="anonymous"></script>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
-    <title>Spotify Clone</title>
-    
-  </head>
-  <body>
-    <div class="sidebar">
-      <div>
-        <img src="logo.png" alt="" class="spotify">
-      </div>
-
-        <div class="navigation">
-          <ul>
-            <li><a href="#" class="home"><i class="fa-solid fa-house icons"></i><span class="nav">Home</span></a></li>
-            <li><a href="#"><i class="fa-solid fa-magnifying-glass icons"></i><span class="nav">Search</span></a></li>
-            <li><a href="#"><i class="fa-solid fa-book icons"></i><span class="nav">Your Library</span></a></li> 
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script
+      src="https://kit.fontawesome.com/39da8831d4.js"
+      crossorigin="anonymous"
+    ></script>
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
+    </style>
 
+    <title>Document</title>
+  </head>
```

Comment:
@Bishakha1589 this indentation should match the title tag. Please check all around your code for this.